### PR TITLE
Add Command and Shell injections test for Java

### PR DESCRIPTION
**Pull request by jandro996**
## Motivation

Test Command and Shell injections in Java

## Changes

- Fix errors in the command injection test file
- Add java endpoints

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/appsec/rasp/test_cmdi.py [hunk 1]
```diff
@@ -18,10 +18,10 @@
 @features.rasp_command_injection
 @scenarios.appsec_rasp
 class Test_Cmdi_UrlQuery:
-    """Shell Injection through query parameters"""
+    """Command Injection through query parameters"""
 
     def setup_cmdi_get(self):
-        self.r = weblog.get("/rasp/cmdi", params={"command": "/bin/evilCommand"})
+        self.r = weblog.get("/rasp/cmdi", params={"command": "/usr/bin/reboot -f"})
```
Comment (NachoEchevarria) [inline tests/appsec/rasp/test_cmdi.py:24]: If a library fails blocking the command execution, would that mean that the host machine is rebooted?

Comment (IlyasShabi) [reply tests/appsec/rasp/test_cmdi.py:24]: You can check my commit https://github.com/DataDog/system-tests/pull/3623/commits/d18269962239311d2ccbfcafecacc34832549275

Comment (jandro996) [reply tests/appsec/rasp/test_cmdi.py]: Aligned with nodejs PR. Thanks!

**Review by NachoEchevarria — APPROVED** [review]
LGTM. Thanks!

**Review by IlyasShabi — APPROVED** [review]
LGTM

**Comment by cbeauchesne** [comment]
Epic PR 🚀  👏  !!

**Pull request merged**